src/js/game/LevelMVC/LevelView.js

```diff
@@ -104,6 +106,7 @@ module.exports = class LevelView {
       "leavesJungle": ["leavesJungle", "Leaves_Jungle0.png", -100, 0],
       "leavesOak": ["leavesOak", "Leaves_Oak0.png", -100, 0],
       "leavesSpruce": ["leavesSpruce", "Leaves_Spruce0.png", -100, 0],
+      "leavesSpruceSnowy": ["leavesSpruceSnowy", "Leaves_SpruceSnowy1.png", -100, 36],
```

Comment:
why not "Leaves_SpruceSnowy0.png"?

Reply:
Same reason as below, I just followed the pattern.
Leaves_Spruce0.png is the first frame referenced on the Spruce png, and Leaves_SpruceSnowy1.png is the first frame referenced on the Spruce Snowy png.